.circleci/config.yml

```diff
@@ -298,3 +327,4 @@ workflows:
       - publish-docker-stratum-bcm-opennsa:
           requires:
             - publish-docker-build
+      - publish-docker-bf-pipeline-builder
```

Comment:
This builds on PRs; need to depend on `publish-docker-build` or filter on `master`

Reply:
Correct, but currently there is no way to know if it works before merging to master.
We'll add the filter before merging.